test/tests/actions/gold/reinforcedBeam.jou

```diff
@@ -0,0 +1,24 @@
+reset
```

Comment:
The mesh files shouldn't be in the gold directory. Just results go there.

Reply:
Corrected!